client/my-sites/site-settings/speed-up-site-settings.jsx

```diff
@@ -95,15 +141,25 @@ class SpeedUpSiteSettings extends Component {
 export default connect( state => {
 	const selectedSiteId = getSelectedSiteId( state );
 	const siteInDevMode = isJetpackSiteInDevelopmentMode( state, selectedSiteId );
+	const siteAcceleratorSupported = isJetpackMinimumVersion( state, selectedSiteId, '6.6-alpha' );
 	const moduleUnavailableInDevMode = isJetpackModuleUnavailableInDevelopmentMode(
 		state,
 		selectedSiteId,
 		'photon'
 	);
+	const photonModuleActive = isJetpackModuleActive( state, selectedSiteId, 'photon' );
+	const assetCdnModuleActive = isJetpackModuleActive( state, selectedSiteId, 'photon-cdn' );
+
+	// Status of the main site accelerator toggle.
+	const siteAcceleratorStatus = !! ( photonModuleActive || assetCdnModuleActive );
 
 	return {
+		assetCdnModuleActive,
```

Comment:
We no longer need to pass the `assetCdnModuleActive` and `photonModuleActive` as connected props, as they're  only used for calculating `siteAcceleratorStatus`.

Let's make sure to remove them from the `propTypes` as well.

Reply:
done in 0723e5029bddd61e1c20cdf7402ad277d9c068c7